newapi/ooniapi/citizenlab.py

```diff
@@ -98,13 +98,14 @@ def update(self, op_code, cur_count, max_count=None, message=""):
 
 
 class URLListManager:
-    def __init__(self, working_dir, github_token, push_repo, origin_repo):
+    def __init__(self, working_dir, github_user, github_token, push_repo, origin_repo):
         self.working_dir = working_dir
         self.origin_repo = origin_repo
         self.push_repo = push_repo
-        self.github_user = push_repo.split("/")[0]
+        self.github_user = github_user
         self.github_token = github_token
         self.repo_dir = self.working_dir / "test-lists"
+        self.push_username = push_repo.split("/")[0]
```

Comment:
Maybe this should be renamed push_orgname?

Reply:
It can be either an org or a username, so it's kind of equivalent which one we choose. I picked the `username` naming for consistency with how it's documented in the github API docs: https://docs.github.com/en/rest/reference/pulls#create-a-pull-request